schemas/registration.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "registration",
+  "description": "Collection of elements required to register the required Donor-Specimen-Sample data to the ARGO Platform. Registration of samples is required before genomic and clinical data submission can proceed.",
+  "key": "submitter_donor_id",
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "gender",
+      "valueType": "string",
+      "description": "Description of the donor self-reported gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Male",
+          "Female",
+          "Other"
+        ]
+      }
+    },
+    {
+      "name": "specimen_submitter_id",
+      "valueType": "string",
+      "description": "Unique identifier of the specimen, assigned by the data provider.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "specimen_type",
+      "valueType": "string",
+      "description": "Tissue source of the biospecimen.",
```

Comment:
based on this description, maybe field name should match it up, ie, `specimen_tissue_source` or something similar?

Reply:
using the terms from @hknahal clinical definition of field.  Hardeep, is there a better or more comprehensive definition for `specimen_type`?

Reply:
Sure, we can change the field name to `specimen_tissue_source`

Reply:
changed!